packages/gatsby/cache-dir/fast-refresh-overlay/components/accordion.js

```diff
@@ -0,0 +1,83 @@
+import * as React from "react"
+import { useId } from "./use-id"
+import * as keys from "../helpers/keys"
+import { match } from "../helpers/match"
+
+function ChevronIcon() {
+  return (
+    <svg
+      focusable="false"
+      preserveAspectRatio="xMidYMid meet"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="currentColor"
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      aria-hidden="true"
+      data-gatsby-overlay="chevron-icon"
+    >
+      <path d="M11 8L6 13 5.3 12.3 9.6 8 5.3 3.7 6 3z" />
+    </svg>
+  )
+}
+
+export function Accordion({ children, ...rest }) {
+  return (
+    <ul data-gatsby-overlay="accordion" {...rest}>
+      {children}
+    </ul>
+  )
+}
+
+export function AccordionItem({
+  children,
+  disabled = false,
+  open = false,
+  title = `title`,
+  ...rest
+}) {
+  const [isOpen, setIsOpen] = React.useState(open)
+  const [prevIsOpen, setPrevIsOpen] = React.useState(open)
+  const id = useId(`accordion-item`)
```

Comment:
We need a stable id here (for both SSR and client for re-hydration) so that's why this helper exists